src/globals.cpp

```diff
@@ -245,6 +245,8 @@ CTweak<uint64_t> pruneIntervalTweak("prune.pruneInterval",
     "How much block data (in MiB) is written to disk before trying to prune our block storage",
     DEFAULT_PRUNE_INTERVAL);
 
+CTweak<uint32_t> netMagic("net.magic", "network prefix override. If 0 use the default", 0);
```

Comment:
As a minor nit, I wonder whether it would make sense to make these docs more explicit, saying which byte in network order corresponds to which byte (as far as I can see: lowest byte is the first byte sent).